sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableServiceAsyncClientTest.java

```diff
@@ -228,12 +227,10 @@ void serviceListTablesWithTopAsync() {
         final String tableName2 = testResourceNamer.randomName("test", 20);
         final String tableName3 = testResourceNamer.randomName("test", 20);
         ListTablesOptions options = new ListTablesOptions().setTop(2);
-        Mono.when(
-            serviceClient.createTable(tableName),
-            serviceClient.createTable(tableName2),
-            serviceClient.createTable(tableName3)
-        ).block(TIMEOUT);
-        
+        serviceClient.createTable(tableName).block(TIMEOUT);
```

Comment:
Is there a reason for splitting this into three sequential blocks rather than running them concurrently?

Reply:
I was having intermittent 429 issues with failures when using `Mono.when` with multiple streams against the CosmosDB API.